yacc/reader.c

```diff
@@ -1459,7 +1476,7 @@ void read_grammar(void)
         }
         at_first = 0;
         if (c == EOF) break;
-        if (isalpha(c) || c == '_' || c == '.' || c == '$' || c == '\'' ||
+        if (In_bitmap(caml_ident_start, c) || c == '_' || c == '.' || c == '$' || c == '\'' ||
                 c == '"')
```

Comment:
The line is too long.

```suggestion
        if (In_bitmap(caml_ident_start, c) || c == '_' || c == '.' || c == '$' ||
                c == '\'' || c == '"')
```